elyra/metadata/schemas/code-snippet.json

```diff
@@ -11,8 +11,7 @@
     },
     "display_name": {
       "description": "The display name of the Code Snippet",
-      "type": "string",
-      "pattern": "^[a-zA-Z][a-zA-Z0-9-_. (){}]*[a-zA-Z0-9)}]$"
+      "type": "string"
```

Comment:
Why is the pattern removed?  The pattern does require to begin with alpha and end with alphanumeric.

Reply:
as it originally was, if i wanted to set the `diplay_name` for a code snippet as "**josé**" or "**test jupyter's kernel**" or "**5 sec delay**" these would all fail.

my understanding was the user should be able to enter any string for `display_name`. if it is necessary to enforce it starts with alpha and end with alphanumeric i can go along that, but do we want to also restrict the rest of the characters in between?

Reply:
Totally correct - that's what that field is for and the previous limitation was completely artificial.  All of the other schemas are this way and they'll need the same change - but the others can be handled separately.  Thanks for noticing that.

Reply:
change https://github.com/elyra-ai/elyra/pull/629/commits/c978ef93f38a3759cc88c10d0cc9b770b659716b has been included in this PR to relax the `display_name` restrictions for other schemas